scipy/spatial/transform/rotation.py

```diff
@@ -195,3 +196,52 @@ def from_dcm(cls, dcm):
             return cls(quat[0], normalized=True)
         else:
             return cls(quat, normalized=True)
+
+    @classmethod
+    def from_rotvec(cls, rotvec):
+        """Initialize class from rotation vector.
+
+        A rotation vector is a 3 dimensional vector which is co-directional to
+        the axis of rotaion and whose norm gives the angle of rotation (in
+        radians).
+
+        Parameters
+        ----------
+        rotvec : array_like, shape (N, 3) or (3,)
+            A single vector or a stack of vectors, where `rot_vec[i]` gives
+            the ith rotation vector.
+        """
+        is_single = False
+        rotvec = np.asarray(rotvec, dtype=float)
+
+        if rotvec.ndim not in [1, 2] or rotvec.shape[-1] != 3:
+            raise ValueError("Expected `rot_vec` to have shape (3,) "
+                             "or (N, 3), got {}".format(rotvec.shape))
+
+        # If a single vector is given, convert it to a 2D 1 x 3 matrix but
+        # set self._single to True so that we can return appropriate objects
+        # in the `as_...` methods
+        if rotvec.shape == (3,):
+            rotvec = rotvec[None, :]
+            is_single = True
+
+        num_rotations = rotvec.shape[0]
+
+        norms = np.linalg.norm(rotvec, axis=1)
+        small_angle = np.nonzero(norms <= 1e-3)[0]
+        large_angle = np.nonzero(norms > 1e-3)[0]
+
+        scale = np.empty(num_rotations)
+        scale[small_angle] = (0.5 - norms[small_angle] ** 2 / 48 +
```

Comment:
A comment with a name or ref for the technique used here (taylor expansion of sin?) would be good. Think about the next dev who has to work with this code, they will probably not immediately know what this is doing